ui_framework/src/components/form/radio/_radio.scss

```diff
@@ -0,0 +1,3 @@
+@include component('kuiRadio') {
```

Comment:
Looks like we're missing some styles here. Do you know why this didn't throw an error since we deleted this mixin?